master/buildbot/reporters/gerrit_verify_status.py

```diff
@@ -226,6 +229,16 @@ def send(self, build):
                     reporter=reporter,
                     category=category,
                     duration=duration)
+
+                # The verify-status plugin will return 204 NO CONTENT
+                # on success.
+                if response.code != http_client.NO_CONTENT:
+                    try:
+                        body = yield response.text()
```

Comment:
not sure why you first try response.text(). For me it does not correspond to any API